css/ampshared.css

```diff
@@ -68,20 +68,25 @@
   position: relative;
 }
 
-.i-amphtml-layout-intrinsic
+.i-amphtml-layout-intrinsic,
+[layout=intrinsic][width][height]:not(.i-amphtml-layout-intrinsic)
```

Comment:
Is `[width][height]` necessary here? They should be implied for valid AMP.

Reply:
I'm not sure why this was done originally, but this explicit list-out-everything matches the other layout selectors. We could probably bring up removing these, to save bytes.